index.html

```diff
@@ -1831,12 +1831,56 @@ <h2>Errors</h2>
 
     <section id="common-constraints-semantic-annotations">
       <h2>Semantic Annotations</h2>
-      <p class="ednote">
-        TODO: Define a set of recommended semantic ontologies for common
-        vocabulary such as
-        <a href="https://github.com/w3c/wot-profile/issues/137">geolocation</a>
-        and <a href="https://github.com/w3c/wot-profile/issues/29">units</a>.
-      </p>
+      <section id="common-constraints-semantic-annotations-geolocation">
+        <h3>Geolocation</h3>
+        <p>
+          <span class="rfc2119-assertion" id="common-constraints-errors-7">
```

Comment:
I think, it is not a good idea to concentrate only on a single geo ontology. The proposal should be open to use different geo location approaches that, e.g., work indoor and outdoor. Building typical uses different location information such as zone, level, room number, etc. . I think many or most IoT devices are used within buildings, so we should support these approaches.